requirements/edx/base.txt

```diff
@@ -152,3 +152,4 @@ analytics-python==0.4.4
 
 # Needed for mailchimp(mailing djangoapp)
 mailsnake==1.6.2
+jsonfield==1.0.3
```

Comment:
From the top of this file:
> DON'T JUST ADD NEW DEPENDENCIES!!!
If you open a pull request that adds a new dependency, you should notify:
* @mollydb - to check licensing
* One of @e0d, @jarv, or @feanil - to check system requirements